src/client/styles/main.scss

```diff
@@ -48,3 +48,28 @@ body {
 li {
   list-style: none;
 }
+
+.fade{
```

Comment:
Space after the `e`.

Reply:
removed the styles for now, until we clarify the animation expectation...